config/config.go

```diff
@@ -0,0 +1,103 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var logger = log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+
+var (
+	version       string
+	configPath    = kingpin.Flag("config.path", "Path of configuration file").Short('c').String()
```

Comment:
Any reason for removing the default `./updog.yaml` config path? I found it helped having the default point to a local file so `make run` would work nicely, as well as running a compiled binary from a UI rather than a shell.

Reply:
I thought the idea was running the test examples during development time, which was better accomplished with updog.test.yaml